README.md

```diff
@@ -1,115 +1,9 @@
-## OpenShift 4.4+ Designs
-Access design documentation for features slotted in the 4.4 release of OpenShift. Click links below for specific section details.
+# OpenShift Designs
+This repository covers designs for the OpenShift Console, which is separated into sections below. Links to OCM designs are available as well.
 
-- [Bulk actions](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/bulk-actions/bulk-actions)
-- [Cron Jobs](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/cron-jobs/cron-jobs)
-- [Diagnostics](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/diagnostics/diagnostics)
-- [Favoriting](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/favoriting/favoriting)
-- [Health Checks](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/health-checks/health-checks)
-- [Search page](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/search/search)
-- [Toolbars](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/toolbars/toolbars)
+- [Administration Console Designs](http://openshift.github.io/openshift-origin-design/admin-console/admin-console)
+- [Developer Console Designs](http://openshift.github.io/openshift-origin-design/dev-console/dev-console)
```

Comment:
Please add the following without links - I can move things around once you have this merged!
- Pipelines
- Serverless

Reply:
Done